operator/pkg/manifest/installer.go

```diff
@@ -352,6 +355,13 @@ func ApplyManifest(componentName name.ComponentName, manifestStr, version string
 	// Base components include namespaces and CRDs, pruning them will remove user configs, which makes it hard to roll back.
 	if componentName != name.IstioBaseComponentName && opts.Prune == nil {
 		opts.Prune = pointer.BoolPtr(true)
+		pwl, ok := componentPruneWhiteList[componentName]
+		if ok {
+			for _, pw := range pwl {
+				pwa := []string{"--prune-whitelist", pw}
```

Comment:
I assume this is additive and the normal stuff like pod is pruned still?

Reply:
yes this is additive